OpenRA.Mods.D2k/Traits/SpiceBloom.cs

```diff
@@ -34,6 +37,9 @@ public class SpiceBloomInfo : ITraitInfo, Requires<RenderSpritesInfo>
 
 		public readonly string ResourceType = "Spice";
 
+		[Desc("Spice blooms only grow on this terrain type.")]
+		public readonly string GrowthTerrainType = "Sand";
```

Comment:
Since we have 1 actor that uses the default and 1 that removes the default, I'd rather we default to "" (or `null`, whichever is appropriate according to our standards) and put "Sand" on the actor that needs it.